<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QMainWindow" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>742</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1008</width>
    <height>742</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1008</width>
    <height>742</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1008</width>
    <height>742</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Home</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QStackedWidget" name="views">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>998</width>
      <height>700</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>17</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="homepage">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>270</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>home</string>
      </property>
     </widget>
     <widget class="QWidget" name="userstoday" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>211</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#userstoday{
   background-color: #D9D9D9;
	border-radius: 12px;
	color:white;	
}</string>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #D9D9D9;
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>User's Today</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>28</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #D9D9D9;
</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="totalsales" native="true">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>211</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#totalsales{
  background-color: #9FFB7E;
border-radius: 12px;
color: rgb(0, 0, 0);
}</string>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #9FFB7E;</string>
       </property>
       <property name="text">
        <string>Total sales</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>28</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #9FFB7E;
</string>
       </property>
       <property name="text">
        <string>123</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="totalorders" native="true">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>10</y>
        <width>221</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#totalorders{
  background-color: #76EBF2;
border-radius: 12px;
color: rgb(0, 0, 0);
}</string>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #76EBF2;</string>
       </property>
       <property name="text">
        <string>Total orders</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>28</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #76EBF2;
</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="namerollno" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>500</y>
        <width>441</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">#namerollno{
  background-color: #9FFB7E;
border-radius: 12px;
color: rgb(0, 0, 0);
}</string>
      </property>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>10</y>
         <width>191</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>18</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #9FFB7E;
font: 18pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Maryam Amin</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>50</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #9FFB7E;
</string>
       </property>
       <property name="text">
        <string>  23p-0052</string>
       </property>
      </widget>
      <widget class="QLabel" name="ruppe">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>70</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#ruppe{
background-color: #37C43C;
border-radius: 10px;
color: rgb(0, 0, 0);
}</string>
       </property>
       <property name="text">
        <string>  Paid  Rs. 200 </string>
       </property>
      </widget>
      <widget class="QLabel" name="labelpic">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>51</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#labelpic{
  background-color: #F2E3D8;
border-radius: 15px;
color: rgb(0, 0, 0);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../Pictures/Screenshots/2024-03-16 (1).png</pixmap>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="tablewidget1">
      <property name="geometry">
       <rect>
        <x>-50</x>
        <y>160</y>
        <width>501</width>
        <height>331</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="tabletTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">#tablewidget1{
  background-color: #F2E3D8;
border-radius: 19px;
color: rgb(0, 0, 0);
}</string>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <column>
       <property name="text">
        <string>#</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Roll No</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Item</string>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="tablewidget2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>150</y>
        <width>561</width>
        <height>511</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#tablewidget2{
  background-color: #C6F3E0;
border-radius: 15px;
color: rgb(0, 0, 0);
}</string>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <column>
       <property name="text">
        <string>#</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Roll No</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Payment</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="orderspage">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>320</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>orders</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tablewidget1_2">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>40</y>
        <width>431</width>
        <height>551</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="tabletTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">#ordertable1{
  background-color: #F2E3D8;
border-radius: 19px;
color: rgb(0, 0, 0);
}</string>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <column>
       <property name="text">
        <string>#</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Roll No</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Item</string>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="orderstable2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>40</y>
        <width>521</width>
        <height>551</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#orderstable2{
  background-color: #C6F3E0;
border-radius: 15px;
color: rgb(0, 0, 0);
}</string>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <column>
       <property name="text">
        <string>#</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Roll No</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Payment</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="foodpage"/>
    <widget class="QWidget" name="userpage"/>
    <widget class="QWidget" name="loanpage"/>
    <widget class="QWidget" name="settingpage"/>
    <widget class="QWidget" name="helppage"/>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>231</width>
      <height>741</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="baseSize">
     <size>
      <width>1980</width>
      <height>1440</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#333333;</string>
    </property>
    <widget class="QPushButton" name="HomeButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>231</width>
       <height>44</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#HomeButton
{
	text-align:left;
	padding-left:20px;
	border:2px solid dodgerblue;
	border-top-left-radius: 20px;
	border-bottom-left-radius: 20px;
	color:white;	
	background:#333333;
}
</string>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
    <widget class="QPushButton" name="OrdersButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>231</width>
       <height>44</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#OrdersButton
{
	text-align:left;
	padding-left:20px;
	color:white;	
	border:none;
	background:#333333;

}</string>
     </property>
     <property name="text">
      <string>Orders</string>
     </property>
    </widget>
    <widget class="QPushButton" name="FoodButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>231</width>
       <height>44</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#FoodButton
{
	text-align:left;
	padding-left:20px;
	color:white;	
	background:#333333;
border:none;
}</string>
     </property>
     <property name="text">
      <string>Food Settings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UserButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#UserButton
{
	text-align:left;
	padding-left:20px;
	color:white;	
	background:#333333;
border:none;
}</string>
     </property>
     <property name="text">
      <string>Users</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LoanButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>231</width>
       <height>44</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#LoanButton
{
	text-align:left;
	padding-left:20px;
	color:white;	
	background:#333333;
border:none;
}</string>
     </property>
     <property name="text">
      <string>Loan Management</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SettingsButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>231</width>
       <height>44</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#SettingsButton
{
	text-align:left;
	padding-left:20px;
	color:white;	
	background:#333333;
border:none;
}</string>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="HelpButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>480</y>
       <width>231</width>
       <height>44</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#HelpButton
{
	text-align:left;
	padding-left:20px;
	color:white;	
	background:#333333;
border:none;
}</string>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>121</width>
       <height>97</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resources/resources.qrc">:/Images/logo.svg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>640</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"> 
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Made with</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>640</y>
       <width>16</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(224, 27, 36);</string>
     </property>
     <property name="text">
      <string>♡  </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>660</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>v 1.0.0</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1008</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menucafekit">
    <property name="title">
     <string>cafekit</string>
    </property>
   </widget>
   <addaction name="menucafekit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
